package-lock.json

```diff
@@ -282,7 +282,7 @@
     "@ckeditor/ckeditor5-adapter-ckfinder": {
       "version": "11.0.4",
       "resolved": "https://registry.npmjs.org/@ckeditor/ckeditor5-adapter-ckfinder/-/ckeditor5-adapter-ckfinder-11.0.4.tgz",
-      "integrity": "sha512-/wpdYrCA3J6MlaqXUUrH/fF5b7g+8Xvlq3bUntTM/GcyeqCFeIJqxumaTIXOuXLobfmVqMKYL1lTrL3WetA2MQ==",
+      "integrity": "sha1-2ylREZQ1Ge+maQ+D1EDtQIOzTjE=",
```

Comment:
Do you know why npm sometimes randomly converts things between sha1 and sha512?

/cc @kevinlee12 -- https://stackoverflow.com/a/51140319 might be relevant.